src/components/ToDoComponent/TodoComponent.tsx

```diff
@@ -1,11 +1,11 @@
-import { Text, View } from 'react-native';
+import { Text, TouchableOpacity, View } from 'react-native';
 import React, { useEffect, useState } from 'react';
 import Card from './Card';
 import data from './Data';
 import { TodoStyles } from './Styles/TodoStyles';
 import Task from './taskType';
 
-const TodoComponent = () => {
+function TodoComponent({ navigationProp }) {
```

Comment:
Nit:
Earlier, in Calendar.tsx you had used the arrow syntax, and here you are using function declaration. It would be better to follow one convention for uniformity.

Reply:
Given that every component is using the Arrow function, I have changed back to the Arrow function only. Could you please check?

Reply:
This is done.